src/pudl/output/pudltabl.py

```diff
@@ -129,6 +131,35 @@ def __init__(
         # Used to persist the output tables. Returns None if they don't exist.
         self._dfs = defaultdict(lambda: None)
 
+        self._register_output_methods()
+
+    def _register_output_methods(self):
```

Comment:
This method gets called from the constructor and will dynamically add methods to get assets from `denorm_assets`. This makes it possible to do:

```py
df = pudltabl.denorm_balance_sheet_assets_ferc1()
```

Under the hood it's just using `get_table_from_db`